google/cloud/spanner/value.cc

```diff
@@ -88,6 +90,52 @@ bool Equal(google::spanner::v1::Type const& pt1,
   }
 }
 
+// A 256-element table mapping all `unsigned char` byte values to a printable
+// string. All naturally printable values are used directly. All non-printable
+// bytes use their octal escape sequence. Whitespace characters use their short
+// form like '\t' and '\n'.
+using ByteType = unsigned char;
+constexpr auto kByteTableSize = std::numeric_limits<ByteType>::max() + 1;
+constexpr std::array<char const*, kByteTableSize> kByteTable = {
+    "\\000", "\\001", "\\002", "\\003", "\\004", "\\005", "\\006", "\\a",
+    "\\b",   "\\t",   "\\n",   "\\v",   "\\f",   "\\r",   "\\016", "\\017",
+    "\\020", "\\021", "\\022", "\\023", "\\024", "\\025", "\\026", "\\027",
+    "\\030", "\\031", "\\032", "\\033", "\\034", "\\035", "\\036", "\\037",
+    " ",     "!",     "\"",    "#",     "$",     "%",     "&",     "'",
```

Comment:
Should not `"\""` be `"\\\""`?  (Another reason why the short-form escapes aren't just whitespace.)

Reply:
Done